packages/serve/src/createConfig.ts

```diff
@@ -1,4 +1,8 @@
 import { devServerOptionsType } from './types';
+// eslint-disable-next-line node/no-extraneous-import
+import { version } from 'webpack-dev-server/package.json';
```

Comment:
Still can be error, I don't think we should use `import`/`require` on the top of files for peer dependencies, I think we don't have a problem here, because we check it before, but better do not do it, because we can refactoring something and break code

Reply:
agree, changing 👍